src/auto-evo/AutoEvoRun.cs

```diff
@@ -407,17 +407,19 @@ protected virtual void GatherInfo(Queue<IRunStep> steps)
         {
             steps.Enqueue(new ModifyExistingSpecies(entry.Value, new SimulationCache(worldSettings), worldSettings,
                 random));
+        }
 
-            steps.Enqueue(new MigrateSpecies(entry.Value, new SimulationCache(worldSettings), random));
+        foreach (var species in allSpecies)
+        {
+            steps.Enqueue(new MigrateSpecies(species, map, worldSettings, new SimulationCache(worldSettings), random));
```

Comment:
Is it absolutely necessary that no migration steps are interspersed between the modify steps? They should not conflict, right?